### PR TITLE
feat: update sats-wagmi with bob-sdk and cleanup

### PR DESCRIPTION
**Pull request by nud3l**
## 📝 Description

- Update sats-wagmi from the latest UI state
- Adds (untested) BOB MM Snap functionality
- Remove the `utils` package
- Remove the `types` package
- Replace `bitcoinjs-lib` in the base adapter with `btc-signer`
- Updates to `bob-sdk` `2.0.0`

### CODE REVIEW

File: packages/react-query/src/prices/usePrices.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  'rocket-pool-eth',
+  'stakestone-ether',
+  'tbtc',
+  'tether',
+  'usd-coin',
+  'wrapped-bitcoin',
+  'wrapped-steth',
+  'bitcoin',
+  'sovryn-dollar'
+];
```
Comment (gregdhill) [inline packages/react-query/src/prices/usePrices.tsx:42]: Ideally these live elsewhere but this is fine for now

File: packages/sats-wagmi/package.json [hunk 2]
```diff
@@ -1,12 +1,11 @@
 {
   "name": "@gobob/sats-wagmi",
-  "version": "0.0.3",
-  "description": "Bob sats-wagmi.",
-  "homepage": "https://github.com/bob-collective/sats-wagmi#readme",
+  "version": "0.0.0",
+  "description": "Bob UI library.",
+  "homepage": "https://github.com/bob-collective/bob-ui#readme",
   "license": "MIT",
```
Comment (gregdhill) [inline packages/sats-wagmi/package.json:6]: Need to change / revert these

File: packages/sats-wagmi/src/connectors/mm-snap.ts [hunk 3]
```diff
@@ -70,17 +87,13 @@ const { ethereum } = window;
 
 const snapId = 'npm:@gobob/bob-snap';
 
+// TODO: distinguish between payment and oridnals address
```
Comment (gregdhill) [inline packages/sats-wagmi/src/connectors/mm-snap.ts:90]:
```suggestion
// TODO: distinguish between payment and ordinals address
```

File: packages/react-query/src/prices/usePrices.tsx (hunk 1 above)
Comment (nud3l) [reply packages/react-query/src/prices/usePrices.tsx:42]: I didn't fully get to clean up the react-query package. It should be in the UI, agree with that.

**Review by gregdhill — APPROVED** [review]

**Pull request merged**